xml/installation-installation-gui_installer.xml

```diff
@@ -238,26 +237,147 @@ controller1,192.168.110.3,b2:72:8d:ac:7c:6f,RACK1,HP-DL360-4PORT,CONTROLLER-ROLE
 myserver4,10.2.10.24,00:14:22:01:23:44,AZ1,,,,
 </screen>
  </section>
-<!-- FIXME: procedure needs to be added here when information received from Cloud 
-  Day0 Squad. Instructions at 
-  https://suse-wiki.dyndns.org/display/ETHOS/SUSE+Manager+Certificate+and+Oneview+Certificate.
-Day0 Squad says that it is critical for this to be included. 
  <section xml:id="discover_servers">
   <title>Optional: Importing Certificates for &susemgr; and &oneview;</title>
   <para>
-   If you intend to use &susemgr;, &oneview; to add servers, those certificates
-   must be accessible to the &install_ui;. Use the following steps to import a
-   &susemgr; certificate.
+   If you intend to use &susemgr; or &oneview; to add servers, certificates for
+   those services must be accessible to the &install_ui;.
   </para>
+  <para>
+   Use the following steps to import a &susemgr; certificate.
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Retrieve the <filename>.pem</filename> file from the &susemgr;.
+    </para>
+<screen>
+ curl -k https://<replaceable>SUSE MANAGER IP address:port</replaceable>/pub/RHN-ORG-TRUSTED-SSL-CERT &gt; <replaceable>PEM_NAME</replaceable>.pem
```

Comment:
* please do: `<replaceable>SUSE_MANAGER_IP</replaceable>:<replaceable>PORT</replaceable>` instead of `<replaceable>SUSE MANAGER IP address:port</replaceable>`, i.e. capitalize it all, keep it short, use underscores, and separate into the two logical replaceables (albeit last one is just a nit)
* also there is a leading space in this line, please remove that.